docs/product/issues/issue-details/replay-issues/index.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Replay Issues
+sidebar_order: 30
+description: "Learn about Session Replay rage click issues."
+---
+
+If you've enabled Session Replay, you'll be able to see rage click issues on the **Issues** page in Sentry. Rage clicks, a series of consecutive clicks on the same unresponsive element, are a sure sign of user frustration and most likely deserve your attention.
+
+![Rage click issue](/product/issues/issue-details/replay-issues/Rage-click-issue-in-issue-stream.png)
+
+## Prerequisites for Seeing Rage Clicks
+
+In order to see rage clicks in your issue stream on the **Issues** page, your organization needs to:
+
+- Be sending [session replay events](/product/session-replay/getting-started/)
+- Enable the JavaScript SDK (or related framework), version 7.60.1 or higher
```

Comment:
I think this is a little confusing because the [javascript SDK docs](https://docs.sentry.io/platforms/javascript/session-replay/) say version 7.27.0 or higher. I'm also not sure if the same version applies to "related framework".

If the requirements are different for session replays and rage click issues, I think it would help to explicitly call that out.

Reply:
Great call! @jas-kas could you weigh in?

Reply:
nit: is "(or related framework)" the phrasing?

something like "(or framework specific bundle)" idk

i can confirm that you need sdk v7.60.1 to get rage clicks.
older versions of the sdk will enable session replay, but you will not get rage-click issues from those.

Reply:
^ +1 what Ryan said. You need to be on a newer SDK version for rage click issues.